README.md

```diff
@@ -4,46 +4,55 @@ The Open Apparel Registry (OAR) is a tool to identify every apparel facility wor
 
 - [Requirements](#requirements)
 - [Setup](#setup)
+  - [Google Maps Platform](#google-maps-platform)
 - [Development](#development)
+  - [Development Data 📈](#development-data-)
   - [Hot Reloading 🔥](#hot-reloading-)
-  - [Development Data](#development-data)
   - [Ports](#ports)
-- [Scripts](#scripts)
+- [Scripts 🧰](#scripts-)
 
 ## Requirements
 
-- Vagrant 2.1+
-- VirtualBox 5.0+
-- AWS CLI 1.1+
-- IAM credentials (for artifacts, secrets, etc)
+- [Vagrant](https://www.vagrantup.com/docs/installation/) 2.1+
+- [VirtualBox](https://www.virtualbox.org/wiki/Downloads) 5.0+
 
 ## Setup
 
-First, configure a local AWS profile with access to an S3 bucket with files containing project specific environment variables:
+Run `setup` to bring up the development environment:
 
 ```bash
-$ aws configure --profile open-apparel-registry
-AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
-AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
-Default region name [None]: eu-west-1
-Default output format [None]:
+./scripts/setup
 ```
 
-Next, use `setup` to bring up the development environment:
+`setup` will provision a virtual machine (VM) that contains the tools needed to get started.
 
-```bash
-$ ./scripts/setup
-```
-
-`setup` will provision a Vagrant VM, using `vboxsf` to mount `./` (relative to `Vagrantfile`) to `/vagrant`, as well as `~/.aws` to `/home/vagrant/.aws`, and execute `update`.
```

Comment:
I thought that this information may not be super relevant to someone who's looking to spin up the project for the first time.